src/aria/widgets/Icon.js

```diff
@@ -206,7 +212,8 @@ module.exports = Aria.classDefinition({
             if (cfg) {
                 var domEvtWrapper;
                 if (domEvent) {
-                    domEvtWrapper = new aria.templates.DomEventWrapper(domEvent);
+                    domEvtWrapper = new aria.templates.DomEventWrapper(domEvent
+                    );
                 }
```

Comment:
There is an issue with your formatter.
Please remove any change which are only about formatting so that we can quickly see real changes.